### PR TITLE
Completed 1- Project setup

### PR DESCRIPTION
**Pull request by Kingsl8yer**
The project setup is completed!

### CODE REVIEW

**Review by B7Bprog — COMMENTED** [review]
Hey Miguel you could leave .env.* as we're gonna have .env.production later on and anyways we don't want to push any .env files to Github.

**Review by pippaamy — APPROVED** [review]
great, go ahead and merge

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
-# Northcoders News API
+###1 You will need to create two .env files for your project: .env.test and .env.development. 
+Into each, add PGDATABASE=, with the correct database name for that environment 
+(see /db/setup.sql for the database names).
```
Comment (B7Bprog) [inline README.md:3]: You could write those database names here to make it easier to set it up and try it.

**Review by B7Bprog — COMMENTED** [review]
Hey Miguel, please see my comment.

**Pull request merged**